docs/getting-started/products/yeth/overview.md

```diff
@@ -0,0 +1,94 @@
+# Overview
+
+**yETH**
+
+yETH is a user-governed liquidity pool token consisting of various Ethereum Liquid Staking Derivatives (LSDs). yETH represents one-to-one [beacon chain ETH](https://ethereum.org/en/upgrades/beacon-chain/) (a.k.a. "ETH staked").
+
+yETH does not accrue any yield, it all goes to st-yETH, which makes yETH a good token to use in a stableswaps like Curve. In order to mint yETH users must provide liquidity for all LSDs currently in the yETH pool, or trade in the yETH<->ETH Curve pool.
+
+**Staked yETH (st-yETH)**
+
+Users can deposit yETH to mint st-yETH, and can burn st-yETH to receive yETH back from the staking contract. Stakers receive all yield and slashings from beacon chain (Ethereum proof-of-stake validators).
+
+By bundling LSDs together, st-yETH generates the best risk-adjusted yields from ETH staking. The protocol management can constantly readjust pool weights in order to maximize yield, while protecting users from catastrophic scenarios if one of the LSDs implodes.
```

Comment:
> By bundling LSDs together, st-yETH generates the best risk-adjusted yields from ETH staking. The protocol management can constantly readjust pool weights in order to maximize yield, while protecting users from catastrophic scenarios if one of the LSDs implodes.

By bundling LSTs together, st-yETH aims to generate the best risk-adjusted yield from ETH staking. Through protocol governance, st-yETH users can readjust pool weights in order to maximize yield, while mitigating  catastrophic scenarios where one or several LSTs in the yETH composition suffer adverse events like de-pegging or security incidents.

Reply:
Maybe this paragraph should be moved to a `*Why yETH?*` section, where we also include other things such as: 

* act as good collateral
* diversify the ecosystem by moving away from large LST protocols and bundling several smaller LST protocols and having them work together

etc. could be in the very top of the doc

Reply:
```suggestion
By bundling LSTs together, st-yETH aims to generate the best risk-adjusted yield from ETH staking. Through protocol governance, st-yETH users can readjust pool weights in order to maximize yield, while mitigating catastrophic scenarios where one or several LSTs in the yETH composition suffer adverse events like de-pegging or security incidents.
```